fenix/app/src/main/java/org/mozilla/fenix/compose/SwipeToDismiss.kt

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+@file:Suppress("DEPRECATION")
+
```

Comment:
Opting to suppress deprecation for two reasons: unblock the update in order to fix bugs that are dependant on this update and because of the fact `AnchoredDraggableState` is still experimental with some support even in the development stage.

The migration will be done in a separate bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1881735